doc/de.po

```diff
@@ -25,9 +25,10 @@ msgid "LINKCHECKER"
 msgstr "LINKCHECKER"
 
 #. type: TH
-#: en/linkchecker.1:1 en/linkcheckerrc.5:1
-#, no-wrap
-msgid "2020-04-24"
+#: en/linkchecker.1:1
+#, fuzzy, no-wrap
+#| msgid "2020-04-24"
+msgid "2020-06-05"
 msgstr "2020-04-24"
```

Comment:
lol

Reply:
wtf

Reply:
Looks like the automated man page translation tool wants to translate everything, including the release date.

(Which maybe makes sense?  Different languages might use different date formats.)

Reply:
i would hope YYYY-MM-DD would be agreed on internationally at this point, but maybe you're right. would still seem crazy to translate that. :)

Reply:
Copied the `2020-06-05` across.